mixins.scss

```diff
@@ -40,197 +43,213 @@
+///     @include x-border-radius(20px 10px);
+///   }
+///
 @mixin x-border-radius ($values) {
 	-webkit-border-radius: $values; // iOS Safari 3.2, Android 2.1
 	        border-radius: $values;
 }
 
-/**
- * @description
- * Generates cross-browser-compatible `box-shadow` for a given element
- *
- * @author drublic
- *
- * @link caniuse http://caniuse.com/css-boxshadow
- * @link spec http://www.w3.org/TR/css3-background/#the-box-shadow
- *
- * @param values
- * @returns
- *   -webkit-box-shadow: <values>;
- *           box-shadow: <values>;
- *
- * @example
- *   .selector {
- *     @include x-box-shadow(5px 5px 10px 5px #aaa);
- *   }
- */
+///
+/// Generates cross-browser-compatible `box-shadow` for a given element
+///
+/// @author drublic
+///
+/// @link http://caniuse.com/css-boxshadow caniuse
+/// @link http://www.w3.org/TR/css3-background/#the-box-shadow spec
+///
+/// @param {String} $values
+///
+/// @output
+/// ```css
+///   -webkit-box-shadow: <values>;
+///           box-shadow: <values>;
+/// ```
+///
+/// @example
+///   .selector {
+///     @include x-box-shadow(5px 5px 10px 5px #aaa);
+///   }
+///
 @mixin x-box-shadow ($values...) {
 	-webkit-box-shadow: $values; // iOS Safari 3.2 - 4.3, Android 2.1+
 	        box-shadow: $values;
 }
 
-/**
- * @description
- * Generates cross-browser-compatible `box-sizing` output for a given element.
- *
- * @author drublic
- *
- * @link caniuse
- * @link spec
- *
- * @param type
- * @returns
- *   -webkit-box-sizing: <type>;
- *      -moz-box-sizing: <type>;
- *           box-sizing: <type>;
- *
- * @example
- *   .selector {
- *     @include x-box-sizing;
- *   }
- */
+///
+/// Generates cross-browser-compatible `box-sizing` output for a given element.
+///
+/// @author drublic
+///
+///
+/// @link http://caniuse.com/#feat=css3-boxsizing caniuse
+/// @link http://www.w3.org/TR/css3-ui/#box-sizing spec
+///
+/// @param {String} $type [border-box]
+///
+/// @output
+/// ```css
+///   -webkit-box-sizing: <type>;
+///      -moz-box-sizing: <type>;
+///           box-sizing: <type>;
+/// ```
+///
+/// @example
+///   .selector {
+///     @include x-box-sizing;
+///   }
+///
 @mixin x-box-sizing ($type: border-box) {
 	-webkit-box-sizing: $type; // Safari <= 5.0, Chrome <= 9.0, iOS Safari 3.2 - 4.3 and Android 2.1 - 3.0
 	   -moz-box-sizing: $type; // FF 2.0 - 28.0
 	        box-sizing: $type; // IE 8, Opera 9.5+
 }
 
-/**
- * @description
- * These mixins generates Media Queries
- *
- * @author drublic
- *
- * @link caniuse http://caniuse.com/#feat=css-mediaqueries
- * @link spec http://www.w3.org/TR/css3-mediaqueries/
- *
- * Note: Please use the CSS unit `em` for device-width in order to stay
- * responsive.
- */
-
-/**
- * @param device-width {number}
- * @content
- * @returns
- *   @media screen and (min-width: <device-width>) {
- *      <content>
- *   }
- *
- * @example
- *   .selector {
- *     x-at-least(40em) { width: 60%; }
- *   }
- */
+///
+/// These mixins generates Media Queries
```

Comment:
Description is always about a single item, hence plural form should not be used.